### DISCUSSION TITLE
求答：如何更改迅雷云盘在alist下的webDAV映射策略（改至302重定向）

### DISCUSSION THREAD

**Discussion by ouclei** [AlistGo/alist#5333] [General]
用alist挂载云盘后，可以使用webDAV映射云盘到本地端，此时因为迅雷云盘的代理策略是本地代理，并无302重定向选项，因此无法在映射网盘中读写盘中内容。

经测试，阿里与百度网盘的302重定向方式是可以在本地端读写的。

求答：如何更改迅雷网盘的代理策略

**Comment by pongfcnkl** [comment]
迅雷网盘只能本地代理，不能302

**Comment by anwen-anyi** [comment]
迅雷云盘只能本地代理或者链接代理，如果想要302需要使用专业版
